content/en/docs/topics/advanced.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Advanced Helm Techniques"
+description: "Explains various advanced features for Helm power users"
+aliases: ["/docs/advanced_helm_techniques"]
+---
+
+This section explains various advanced features and techniques for using Helm.
+The information in this section is intended for "power users" of Helm that wish
+to do advanced customization and manipulation of their charts and releases. Each
+of these advanced features comes with their own tradeoffs and caveats, so each
+one must be used carefully and with deep knowledge of Helm. Or in other words,
+remember the [Peter Parker
+principle](https://en.wikipedia.org/wiki/With_great_power_comes_great_responsibility)
+
+## Post Rendering
+Post rendering gives chart installers the ability to manually manipulate,
+configure, and/or validate rendered manifests before they are installed by Helm.
+This allows users with advanced configuration needs to be able to use tools like
+[`kustomize`](https://kustomize.io) to apply configuration changes without the
+need to fork a public chart or requiring chart maintainers to specify every last
+configuration option for a piece of software. There are also use cases for
+injecting common tools and side cars in enterprise environments or analysis of
+the manifests before deployment.
+
+### Prerequisites
+- Helm 3.1+
+
+### Usage
+A post-renderer can be any executable that accepts rendered Kubernetes manifests
+on STDIN and returns valid Kubernetes manifests on STDOUT. It should return an
+non-0 exit code in event of a failure. This is the only "API" between the two
+components. It allows for great flexibility in what you can do with your
+post-render process.
+
+A post renderer can be used with `install`, `upgrade`, and `template`. To use a
+post-renderer, use the `--post-renderer` flag with a path to the renderer
+executable you wish to use:
+
+```shell
+$ helm install mychart stable/wordpress --post-renderer ./path/to/executable
+```
+
+If the provided path does not contain any separators, it will search first in
+the Helm plugins directory, then in `$PATH`, otherwise it will resolve any
+relative paths to a fully qualified path. Searching in the Helm plugins
+directory allows you to leverage the plugins system for distributing your
+executables within a team or company.
+
+If you wish to use multiple post-renderers, call all of them in a script or
+together in whatever binary tool you have built. In bash, this would be as
+simple as `renderer1 | renderer 2 | renderer3`.
+
+You can see an example of using `kustomize` as a post renderer
+[here](https://github.com/thomastaylor312/advanced-helm-demos/tree/master/post-render).
+
+### Caveats
```

Comment:
Hugo, what we use for docs, has the ability to do notices, warnings, etc. The docs on that are at https://learn.netlify.com/en/shortcodes/notice/. Should we do that here?

Reply:
Oh, that's nice. I'll add that in

Reply:
Looks like we don't have support for it (or something else is up):
```
Error: Error building site: "/opt/build/repo/content/en/docs/topics/advanced.md:57:1": failed to extract shortcode: template for shortcode "notice" not found
```

Going to remove for now, but if we find a way to make it work (cc @flynnduism in case you have any tips. No need to spend lots of time on it if it isn't easy), I'll add it back